test/test_helper.rb

```diff
@@ -50,8 +50,8 @@ def postgres?
 end
 
 StrongMigrations.add_check do |method, args|
-  if method == :add_foreign_key && args[0].to_s == "users"
-    stop! "No foreign keys on the users table"
+  if method == :add_column && args[1].to_s == "forbidden"
+    stop! "Cannot add forbidden column"
```

Comment:
Went ahead and changed the custom example to keep the test suite happy.

Reply:
👍